HUD/N_0xf83d0febe75e62c9.md

```diff
@@ -5,20 +5,27 @@ ns: HUD
 
 ```c
 // 0xF83D0FEBE75E62C9
-void _0xF83D0FEBE75E62C9(Any p0, Any p1, Any p2, Any p3, Any p4, Any p5, Any p6, Any p7);
+void _0xF83D0FEBE75E62C9(Blip blip, Any p1, Any p2, float width, Any p4, float length, float heading, Any p7);
 ```
 
 ```
 NativeDB Added Parameter 9: Any p8
 NativeDB Introduced: v1290
 ```
 
+Sets the 'data' for a cone for a blip
+
+See [here](https://docs.fivem.net/docs/game-references/hud-colors/) for the hud colours for the cone
+
+Name in scripts: `SETUP_FAKE_CONE_DATA()`
+
 ## Parameters
-* **p0**:
-* **p1**:
-* **p2**:
-* **p3**:
-* **p4**:
-* **p5**:
-* **p6**:
-* **p7**:
+* **blip**: A blip from an entity, for example ch_prop_ch_cctv_cam_01a
+* **p1**: -1.0 in decompiled scripts (unk)
+* **p2**: 1.0 in decompiled scripts (unk)
+* **width**: Width of the cone (0.0 to ~3.0) 
+* **p4**: 1.0 in decompiled scripts (unk)
+* **length**: Length of the cone. (8.2 is 'normal' sized
+* **heading**: Not your typical heading. Can be calculated with (heading + 180.00) * 0.017453292
+* **p7**: 1 in decompiled scripts (unk)
+* **p8**: Hud colour 
```

Comment:
```
ParseError: A parameter definition was found for a parameter that does not exist. Parameter name: p8
```

At the time, new parameters cannot be added, if they were indeed added to the native method then they should be defined in the native description, look at the following native method for an example:

https://github.com/citizenfx/natives/blob/e7ffedc4a33afe3a68600a649a4f641442217478/VEHICLE/GetVehicleHealthPercentage.md?plain=1#L13

Thank you for your contribution nonetheless!

Reply:
I’ll fix this as soon as I can

Reply:
For niche[1] undocumented natives like this, this is generally not required.

`Any` would be mapped to `int`, so if a server is successfully using the native definition (without manually invoking), they are jumping through some serious hoops. And in the infinitely small case of that being untrue, it should serve as punishment for not documenting their research.

[1] The rule of thumb I used in the past was: undocumented with more complex types or requirements (e.g., the value needs to be non-zero so false/0/0.0, etc., conflicting is meaningless).

...

Radians are a universal unit, including the formula for converting between degrees and radians is superfluous ('... in radians' as a parameter description should be sufficient). Secondly, [Tau](https://tauday.com/tau-manifesto) is superior.